telemetry/definitions/commonDefinitions.json

```
@@ -5971,6 +5988,24 @@
                 }
             ]
         },
+        {
+            "name": "languageServer_setup",
+            "description": "LSP setup event",
```

the namespace can be "lsp". then each mertic can say if it's clien-specific or server-specific, assuming that is something the metric wants to call out.

```suggestion
            "name": "lsp_setupServer",
            "description": "Started an LSP server",
```